evennia/contrib/actions/setup.py

```diff
@@ -0,0 +1,20 @@
+from evennia import ObjectDB
```

Comment:
I cannot put setup in utils.py without changing isinstance(self, ActionCharacter) etc. to isinstance (self, DefaultCharacter) etc, as importing things from evennia.contrib.actions.typeclasses.py into utils.py will otherwise create import errors. I don't find this surprising, as typeclasses.py imports from handlers.py, which imports from utils.py as well as from actions.py, which in turn imports from utils.py. If there is any way around this, please let me know.